qcodes/instrument/base.py

```diff
@@ -415,7 +415,7 @@ class Instrument(InstrumentBase, AbstractInstrument):
 
     _all_instruments: "Dict[str, weakref.ref[Any]]" = {}
```

Comment:
here as well or not (i don't remember)
```suggestion
    _all_instruments: "Dict[str, weakref.ref[Instrument]]" = {}
```